gradle/verification-metadata.xml

```diff
@@ -507,6 +507,11 @@
             <pgp value="47504b76cf89c15c0512d9afe16ab52d79fd224f"/>
          </artifact>
       </component>
+      <component group="com.google.code.gson" name="gson" version="2.8.6">
+         <artifact name="gson-2.8.6.jar">
+            <pgp value="afcc4c7594d09e2182c60e0f7a01b0f236e5430f"/>
```

Comment:
Checking this one, just to be sure.

Reply:
Just had a look, as the UID was a bit weird.
The packet is:
```
# off=0 ctb=c6 tag=6 hlen=3 plen=269 new-ctb
:public key packet:
	version 4, algo 1, created 1448391204, expires 0
	pkey[0]: [2048 bits]
	pkey[1]: [17 bits]
	keyid: 7A01B0F236E5430F
# off=272 ctb=cd tag=13 hlen=2 plen=43 new-ctb
:user ID packet: "Inderjeet Singh <inder@alumni.stanford.edu>"
# off=317 ctb=c2 tag=2 hlen=3 plen=312 new-ctb
:signature packet: algo 1, keyid 7A01B0F236E5430F
	version 4, created 1448391204, md5len 0, sigclass 0x13
	digest algo 2, begin of digest ea bf
	hashed subpkt 2 len 4 (sig created 2015-11-24)
	hashed subpkt 27 len 1 (key flags: 03)
	hashed subpkt 11 len 5 (pref-sym-algos: 9 8 7 3 2)
	hashed subpkt 21 len 5 (pref-hash-algos: 8 2 9 10 11)
	hashed subpkt 22 len 3 (pref-zip-algos: 2 3 1)
	hashed subpkt 30 len 1 (features: 01)
	hashed subpkt 23 len 1 (keyserver preferences: 80)
	subpkt 16 len 8 (issuer key ID 7A01B0F236E5430F)
	data: [2045 bits]
# off=632 ctb=ce tag=14 hlen=3 plen=269 new-ctb
:public sub key packet:
	version 4, algo 1, created 1448391204, expires 0
	pkey[0]: [2048 bits]
	pkey[1]: [17 bits]
	keyid: C3E640F38D845FA2
# off=904 ctb=c2 tag=2 hlen=3 plen=287 new-ctb
:signature packet: algo 1, keyid 7A01B0F236E5430F
	version 4, created 1448391204, md5len 0, sigclass 0x18
	digest algo 2, begin of digest 6c 63
	hashed subpkt 2 len 4 (sig created 2015-11-24)
	hashed subpkt 27 len 1 (key flags: 0C)
	subpkt 16 len 8 (issuer key ID 7A01B0F236E5430F)
	data: [2048 bits]
```
The ASC file, the keyid agrees, and everything checks out, so this is good to go.